pyvista/plotting/plotting.py

```diff
@@ -4585,6 +4594,31 @@ def export_obj(self, filename):
         exporter.SetRenderWindow(self.ren_win)
         return exporter.Write()
 
+    @property
+    def _datasets(self):
+        """List of all datasets associated with this plotter."""
+        datasets = []
+        for renderer in self.renderers:
+            for actor in renderer.actors.values():
+                mapper = actor.GetMapper()
+
+                # ignore any mappers whose inputs are not datasets
+                if hasattr(mapper, 'GetInputAsDataSet'):
+                    datasets.append(mapper.GetInputAsDataSet())
+
+        return datasets
```

Comment:
This only seems to be used in a test, so I assume this is for some other work :P

Reply:
There have been times in the past where this would have been useful as a convenience method. This is indeed used in a closed source project; maybe others will find it useful.